_ml-commons-plugin/prebuilt-models.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Pre-trained models
```

Comment:
This is confusing - are they pre-built models as the name of the doc implies, or are they pre-trained models like the title and content implies? They should be consistent.

Reply:
Agree, about the comment on consistency. The industry standard term is pretrained models.